ignite/engine/engine.py

```diff
@@ -227,12 +235,12 @@ def print_epoch(engine):
         # print_epoch handler is now unregistered
     """
 
-    def __init__(self, event_name, handler, engine):
+    def __init__(self, event_name: str, handler: Callable, engine: Engine):
```

Comment:
Probably, I think `event_name` is not `str` here, but `EventWithFilter`, `CallableEvents` or `Enum`.

Reply:
Done